pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -97,6 +97,7 @@ func (pl *CSILimits) Filter(ctx context.Context, _ *framework.CycleState, pod *v
 	if err != nil {
 		// TODO: return the error once CSINode is created by default (2 releases)
 		klog.V(5).InfoS("Could not get a CSINode object for the node", "node", klog.KObj(node), "err", err)
+		return framework.AsStatus(err)
```

Comment:
This will cause the scheduler/cluster-autoscaler to retry immediately, as opposed to returning an UnschedulableAndUnresolvable status.

Is retrying the expected behavior?

Reply:
I think the comment is incorrect as discussed in https://github.com/kubernetes/kubernetes/pull/107787#discussion_r1080559721.

Autoscaler doesn't simulate CSINode objects today, so we can't return an error otherwise autoscaler will fail forever.

Reply:
Oh wait, sorry, this is my miss, I return the error here to checkout the logs I added in the test. But I only removed the logs but forgot to remove this line.